src/System.Data.Extensions/src/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp-Windows_NT;
```

Comment:
As mentioned, I would think you could remove all configurations in all your configurations.props leaving only `netstandard`. There is nothing Unix/Windows specific, and probably nothing netcoreapp specific.